src/components/CreateUserForm/index.tsx

```diff
@@ -0,0 +1,138 @@
+import { useForm, SubmitHandler } from 'react-hook-form';
+
+type FormValues = {
+  firstName: string;
+  lastName: string;
+  role: string;
+  phoneNumber: number;
+  email: string;
+};
+
+const CreateUserForm = () => {
+  const { register, handleSubmit } = useForm<FormValues>();
+
+  const onSubmit: SubmitHandler<FormValues> = async (data) => {
+    const BASE_URL = process.env.ITEZ_API_URI;
+
+    try {
+      const res = await fetch(`${BASE_URL}/user`, {
```

Comment:
just curious are we using SWR here?
https://digiandcsh.slack.com/archives/C02JKGD1550/p1637114904084500